### ISSUE TITLE
Check AUC-PR calculation is correct

### ISSUE THREAD

**Issue by ddomingof** [pathwayforte/pathway-forte#19]
https://github.com/pathwayforte/pathway-forte/blob/e34324a873ad817ea301ce282b4e31bc27c54506/src/pathway_forte/prediction/binary.py#L143-L145

@aldisirana would you please check this

**Comment by AldisiRana** [comment]
This is the metric I've been using for AUC-PR.
sklearn.metrics.average_precision_score()
read more about it here: https://www.w3cschool.cn/doc_scikit_learn/scikit_learn-modules-generated-sklearn-metrics-average_precision_score.html
